hudi-client/pom.xml

```diff
@@ -102,6 +102,12 @@
       <artifactId>spark-sql_${scala.binary.version}</artifactId>
     </dependency>
 
+    <!-- spark-avro -->
+    <dependency>
```

Comment:
we have moved to spark-avro in apache spark IIRC. we should not use com.databricks anymore?

Reply:
This package is being used only in the unit tests for the function createConverterToSQL to convert Avro record to a spark sql row. Is there another way?

I have added the test-scope to this dependency.